RELEASE_NOTES.md

```diff
@@ -34,4 +34,23 @@ not designed to be a general solution to performance issues inside of chef-clien
 
 This debugging feature will mostly be useful to people who are already Ruby experts.
 
+### New ksh resource
 
+ksh is now available to use as a resource and is available as an interpreter in the script resource.
+
+Please see the DOC_CHANGES for more details.
+
+## Better handling of Windows service logging
```

Comment:
can we be more clear that this is all about the chef client service, not necessarily a service managed with `windows_service` ?